Test/Mono.Cecil.Tests/CompilationService.cs

```diff
@@ -28,11 +28,13 @@ public CompilationResult (DateTime write_time, string result_file)
 	}
 
 	public static class Platform {
-
+#if NET_CORE
+		public static bool OnMono => Environment.OSVersion.Platform == PlatformID.Unix || Environment.OSVersion.Platform == PlatformID.MacOSX;
```

Comment:
We should use Portable Pdb to make the pdb test work https://github.com/dotnet/core/blob/master/Documentation/diagnostics/portable_pdb.md